.circleci/config.yml

```diff
@@ -48,11 +48,13 @@ jobs:
       - run: 
           name: Pull Deps
           command: |
+            set -x
             source ${INIT_ENV}
-            ./src/CARMASscInterfaceWrapper/docker/checkout.sh ${PWD}
+            ./src/CARMASscInterfaceWrapper/docker/checkout.bash -r ${PWD}
       - run:
           name: Build Driver
           command: |
+            set -x
```

Comment:
The set -x commands here and above seem pretty pointless and are not in the other ci setups so lets remove them.